core/components/atoms/statusHints/StatusHints.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from 'react';
+import Text from '@/components/atoms/text';
+import classNames from 'classnames';
+
+export type Appearance = 'info' | 'alert' | 'warning' | 'success' | 'default';
+
+export interface StatusHintsProps {
+  /**
+   * Describes label of the `StatusHints`
+   */
+  label: string;
```

Comment:
@riyalohia  Instead of taking label as an attribute, please use children prop to render it similar to `Text` and `Label` components.